airflow/providers/google/cloud/operators/cloud_build.py

```diff
@@ -175,21 +180,34 @@ class CloudBuildCreateOperator(BaseOperator):
     """
 
     template_fields = ("body", "gcp_conn_id", "api_version")  # type: Iterable[str]
+    template_ext = ['.yml', '.yaml', '.json']
 
     @apply_defaults
     def __init__(self,
-                 body: dict,
+                 body: Union[dict, str],
```

Comment:
Can you also update the docstring?  There should be information that the request body or file path can be passed as a parameter.

Reply:
Your right, I missed it. It is added now